2.4/Dockerfile

```diff
@@ -39,8 +24,8 @@ RUN apt-get update \
 		libaprutil1-dev \
 		liblua5.2-0 \
 		libnghttp2-14=$NGHTTP2_VERSION \
-		libpcre++0 \
-		libssl1.0.0=$OPENSSL_VERSION \
+		libpcre++0v5 \
+		libssl1.1=$OPENSSL_VERSION \
 		libxml2 \
```

Comment:
It's not something we should do for _this_ PR, but I think in the future we probably ought to consider something like https://github.com/docker-library/php/blob/32575d71484b97b6a870527953c986a4c076b99b/Dockerfile-debian.template#L185-L196 so we don't have to hand-maintain this list (which is admittedly fragile and error-prone).